pkg/api/types.go

```diff
@@ -2213,6 +2213,17 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "SYSTEM" is a special keyword
+	// which indicates the highest priority. Any other name must be defined by
+	// creating a PriorityClass object with that name.
+	// If not specified, the pod priority will will be default or zero if there is
```

Comment:
The plan to have the default in the first version. We will definitely have the ability to define a "global" default priority. In the future versions, we may add per namespace default as well.